### DISCUSSION TITLE
Glide Typing Still Works v0.4.3

### DISCUSSION THREAD

**Discussion by LinuxinaBit** [florisboard/florisboard#2721] [General]
\**evil laughter\**

https://github.com/user-attachments/assets/74ba7dbd-3736-43d9-9a41-c011d19a4f34

I finally worked up the courage to update to v0.4.3 from v0.3.16 and lose glide typing support, but it turns out glide typing is still around, even after a complete phone restart.
This is a miracle!

**Comment by Gibbz** [comment]
Lucky! Didn't work for me

**Comment by BHydden** [comment]
The function was never removed, just the settings toggle to turn it off/on. If you turn it on before updating, it will remain on. This is repeatable, just install an older version, toggle it on, then update to latest. 👍 no idea why it was done this way but here we are 🤷‍♂️

**Comment by enxio27** [comment]
Still working in v0.5.1, but noticeably degraded.  It seems to get lost after about 5-6 characters now.  😥  This is one of my most-used features, so I REALLY hope the devs bring it back soon!
